src/index.html

```diff
@@ -5,9 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main>
+
+      <article class="card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Image of APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+        <div class="card__title">
+          <p>APPLE A1419 iMac 27" Retina </p>
+          <p>5K Monoblock (MNED2UA/A)</p>
+        </div>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__reviews card__reviews--text">
+            Reviews: 5
+          </p>
+        </div>
+        <div class="card__price">
+          <div class="card__price card__price--text">
+            Price:
+          </div>
+          <div class="card__price card__price--num">
+            $2,199
+          </div>
+        </div>
+        <button class="card__button" data-qa="hover">
+          BUY
+        </button>
+      </article>
+
+    </main>
```

Comment:
`card__price--num` is not informative, replace it
```suggestion
          </p>
        </div>
        
        <div class="card__price">
          <div class="card__price card__price--text">
            Price:
          </div>
          
          <div class="card__price card__price--num">
            $2,199
          </div>
        </div>
        
        <button class="card__button" data-qa="hover">
          BUY
        </button>
      </article>
    </main>
```